### ISSUE TITLE
Host base template(s) in a manner compatible with `qvm-template`

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-workstation#890]
We currently use a single repo (https://yum.securedrop.org/workstation/dom0/f32/) to distribute both the SecureDrop Workstation installer RPM, and the base template. Our configuration seems seems to be causing issues with `qvm-template` -- once the template is installed, we see errors such as `Unexpected data format` on `qvm-template` subcommands like `qvm-template list`. We should ensure that we distribute templates in a manner that Qubes expects, which may entail doing so through a separate repository.

**Comment by eloquence** [comment]
We have at least one case where a user's `fedora-37` install is failing with "Template not found". This could be related to the template database issue (they are also getting the "Unexpected data format" error on `qvm-template list`); the required repo appears to be present and configured.

**Comment by eloquence** [comment]
My read on the `qvm-template` repo parsing issue is as follows:

- Yes, hosting the template in a dedicated template repo would resolve that issue and would probably be cleaner to avoid any problematic interactions between template RPMs and non-template RPMs in future. 

- However, per `qvm-template`'s own code comments, it should not be required. The `qvm-template` tool is already written to ignore packages that are not templates:
https://github.com/QubesOS/qubes-core-admin-client/blob/6685193618ea127bf173f714872ac0030be61339/qubesadmin/tools/qvm_template.py#L558

- The reason this isn't working is due to a bug in `qvm-tempalte` that is marked as a `FIXME` - its parsing code does not handle descriptions that start with a newline:
https://github.com/QubesOS/qubes-core-admin-client/blob/6685193618ea127bf173f714872ac0030be61339/qubesadmin/tools/qvm_template.py#L544

- Our published `securedrop-workstation-dom0-config` RPM _does_ have exactly such a description, as you can verify by running `rpm -qi` on it. Because it's hosted in the same repo as the template, this causes `qvm-template` to choke.

<details>
<summary>
Click to see details
</summary>

```
Name        : securedrop-workstation-dom0-config
Version     : 0.6.1
Release     : 1.fc25
Architecture: noarch
Install Date: (not installed)
Group       : Library
Size        : 324704
License     : GPLv3+
Signature   : RSA/SHA256, Wed 01 Jun 2022 05:39:42 PM PDT, Key ID 188edd3b7b22e6a3
Source RPM  : securedrop-workstation-dom0-config-0.6.1-1.fc25.src.rpm
Build Date  : Wed 01 Jun 2022 02:41:22 PM PDT
Build Host  : securedrop-workstation-dom0-config
Relocations : (not relocatable)
URL         : https://github.com/freedomofpress/securedrop-workstation
Summary     : SecureDrop Workstation
Description :

This package contains VM configuration files for the Qubes-based
SecureDrop Workstation project. The package should be installed
in dom0, or AdminVM, context, in order to manage updates to the VM
configuration over time.
```
</details>

- However, if you build a version from `main` it does not have this issue, because @eaon has already fixed it in a6383308d165db1b11227163b35670227a77982b , which removed the extra newline :-)

- In other words, if we simply publish a new RPM (and/or split the repo),  this should at least fix the parsing issue; it's unclear to me still if this issue is related to #813 or the user-reported issue of `fedora-37` not being installable. But still, making sure that `qvm-template` behaves well with our RPMs should make it easier to reason about other issues.

**Comment by rocodes** [comment]
Plans to stop shipping a custom SDW template should close this issue, assuming the repo settings are fine for the sdw dom0 rpm - see https://github.com/freedomofpress/securedrop-workstation/issues/765#issuecomment-1813308727

**Comment by rocodes** [comment]
Closing, see #969
